2022/incoming/the-delivery-map.pod

```diff
@@ -0,0 +1,149 @@
+
+It was already a nice help for Santa, but the elf wanted to make it a bit better.
+
+First, to materialize the direction using C<< directed => 1 >> and change the node shape to house C<< shape => "house" >> because after all, these families live in houses!
+
+=begin perl
+
+#!/usr/bin/env perl
+
+use GraphViz2;
+
+my $graph = GraphViz2 -> new(
+	global => {name => 'Birmingham', directed => 1},
+	graph  => {rankdir => 'LR'},
+);
+
+$graph->default_node(
+	shape    => "house",
+	width    => "2",
+	height   => "2",
+);
+
+$graph -> add_edge(from =>  "Family Anderson", to =>  "Family Jones");
+$graph -> add_edge(from =>  "Family Jones", to =>  "Family Taylor");
+$graph -> add_edge(from =>  "Family Taylor", to =>  "Family Williams");
+
+$graph->run(format => 'svg', output_file => 'better.svg');
+
+=end perl
+
+And got this result:
+
+=begin html
+
+<center><img src="../share/static/better.svg" alt="A better delivery map"></center>
+
+=end html
+
+=head2 The full procedure for Santa
+
+Santa was already happy with this map but if was thinking about disconnecting a bit more. 
+
+So if it would be possible to make it appear also the "preparation" steps, it would be perfect for Santa!
+
+To give you details about Santa's typical day: he leaves his home, go to supermarket for chocolate's bars to sustain the travel, walks to the sheds' garage and finally starts the tour:
+
+=begin perl
+
+#!/usr/bin/env perl
+
+use GraphViz2;
+
+my $graph = GraphViz2 -> new(
+	global => {name => 'Birmingham', directed => 1},
+	graph  => {rankdir => 'LR'},
+);
+
+$graph->default_node(
+	shape    => "house",
+	width    => "2",
+	height   => "2",
+);
+
+$graph -> push_subgraph(name => 'cluster_NorthPole', graph => {
+	label => 'North Pole',
+});
+$graph -> add_node(name => "Santa's house");
+$graph -> add_node(name => "Supermarket");
+$graph -> add_node(name => "Sleds' garage");
+$graph -> pop_subgraph;
+
+
+$graph -> push_subgraph(name => 'cluster_Birmingham', graph => {
+	label => 'Birmingham',
+});
+$graph -> add_node(name => "Family Anderson");
+$graph -> add_node(name => "Family Jones");
+$graph -> add_node(name => "Family Taylor");
+$graph -> add_node(name => "Family Williams");
+$graph -> pop_subgraph;
+
+$graph -> add_edge(from =>  "Santa's house", to =>  "Supermarket");
+$graph -> add_edge(from =>  "Supermarket", to =>  "Sleds' garage");
+$graph -> add_edge(from =>  "Sleds' garage", to =>  "Family Anderson");
+$graph -> add_edge(from =>  "Family Anderson", to =>  "Family Jones");
+$graph -> add_edge(from =>  "Family Jones", to =>  "Family Taylor");
+$graph -> add_edge(from =>  "Family Taylor", to =>  "Family Williams");
+
+$graph->run(format => 'svg', output_file => 'final.svg');
+
+=end perl
+
+It resulted in this graph:
+
+=begin html
+
+<center><img src="../share/static/final.svg" alt="Final delivery map" width='1200'></center>
+
+=end html
+
+Elves printed such kind of maps for every deliveries Santa has to do, and Christmas was once again a success!
```

Comment:
The elves started to print these maps for all the deliveries Santa had to make, and ...